docs/docs/guides/e2e/02-data-transfer.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Data Transfer
+tags:
+  - testing
+  - e2e
+  - data-transfer
+  - infrastructure
+---
+
+## Overview
+
+This document looks particularly at the `@strapi/data-transfer` package framed around how we use it with our test instances in the e2e test suite. It is not a comprehensive explanation of how to use `@strapi/data-transfer`. For that information you should [view the documentation](https://docs.strapi.io/developer-docs/latest/developer-resources/data-management.html) surrounding it.
+
+## Why use Data Transfer?
```

Comment:
Some issues with using Data Transfer:

Maintaining the .tar files is going to become a nightmare. It will need to be done after every schema change, or every time we want additional data in the dataset.

So, we will need a script as soon as possible to regenerate the datasets automatically, similar to the front-end regenerating the snapshots.

And in doing THAT, it will be difficult to see a diff between tars when updated, and can't be viewed on github. So we may also need to add a folder provider for DTS (which should be quite simple, actually) to allow us to have unarchived files to track and review changes to them.

Reply:
> Maintaining the .tar files is going to become a nightmare. It will need to be done after every schema change, or every time we want additional data in the dataset.
> So, we will need a script as soon as possible to regenerate the datasets automatically, similar to the front-end regenerating the snapshots.

Yep you're probably right, but at the moment it was the easiest option we had to move forward with the limited bandwidth.